pkg/diff/diff_test.go

```diff
@@ -871,7 +871,7 @@ func TestStructuredMergeDiff(t *testing.T) {
 		// then
 		require.NoError(t, err)
 		assert.NotNil(t, result)
-		assert.False(t, result.Modified)
+		assert.True(t, result.Modified)
```

Comment:
Why does this result in a diff when it didn't before?

Reply:
I've updated the deployment to actually have some changes, to better replicate the behavior from the bug report.